site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java

```diff
@@ -132,6 +136,8 @@ private HstQuery buildQuery(HstRequest request) {
                     constraint(".").contains(queryIncWildcards)
                 )
             ))
+            .limit(10)
+            .offset(getCurrentPage(request) * 10)
```

Comment:
Is the "10" the same pageSize configurable value in (search.yaml - /hst:hst/hst:configurations/common/hst:pages/search/main) or is it used for something else?  If same, the SearchComponent uses paramInfo.getPageSize() to retrieve this value, so could use this instead?

Reply:
Have removed this as discussed with @wojtek-oledzki